src/Illuminate/Support/Arr.php

```diff
@@ -361,6 +361,26 @@ public static function isAssoc(array $array)
         return array_keys($keys) !== $keys;
     }
 
+    /**
+     * Determines if keys of the provided array exist in another array.
+     *
+     * @param  array  $needed
+     * @param  array  $provided
+     * @return bool
+     *
+     * @throws \InvalidArgumentException
+     */
+    public static function keysExists(array $needed, array $provided)
```

Comment:
I think you should remove the final `s` from `exists`

Reply:
Maybe we can keep the conversation going at a single place (in the first thread above) ?